PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+(brief description / PR title)
+
+** Don't forget to label this PR as either: Open for visibility or Ready for review! **
+
+JIRA: **(link to JIRA ticket)**
+Linked PRs: **(links to corresponding PRs, optional)**
+
+## Changes
+- (change1)
+
+## How to test-drive this PR
+- (necessary config changes)
+- (necessary corresponding PRs)
+- (how to access the new / changed functionality -- fixtures, URLs)
+
+## TODOS:
+- [ ] Update tutorial documentation in the `dev` folder to match the modified tutorial.
```

Comment:
Should we specify that these are the docs in the `astro` repo? Or is it obvious given that this repo doesn't have a `docs` folder?